recipes/supervisor/post-link.sh

```diff
@@ -0,0 +1,6 @@
+if [[ $(id -u) == 0 ]]; then
+    mkdir -p /etc/supervisord/ && ln -s $PREFIX/etc/supervisord/ /etc/supervisord/
+    ln -s $PREFIX/etc/supervisord.conf /etc/supervisord.conf
```

Comment:
No, actually it's not.  The post-install shouldn't be doing this linking if you're not running as root.  So if you've installed miniconda as root, then it'll link for you.  If not, then you just get your file structure in your $PREFIX.